crates/cdk-sqlite/src/wallet/mod.rs

```diff
@@ -1255,4 +1258,75 @@ mod tests {
         assert_eq!(retrieved_dleq.s.to_string(), s.to_string());
         assert_eq!(retrieved_dleq.r.to_string(), r.to_string());
     }
+
+    #[tokio::test]
+    async fn test_mint_quote_operations() {
+        use std::str::FromStr;
+
+        use cdk_common::mint_url::MintUrl;
+        use cdk_common::wallet::MintQuote;
+        use cdk_common::{Amount, CurrencyUnit, MintQuoteState, PaymentMethod};
+
+        // Create a temporary database
+        let path = std::env::temp_dir()
+            .to_path_buf()
+            .join(format!("cdk-test-mint-quotes-{}.sqlite", uuid::Uuid::new_v4()));
+
+        #[cfg(feature = "sqlcipher")]
+        let db = WalletSqliteDatabase::new(path, "password".to_string())
+            .await
+            .unwrap();
+
+        #[cfg(not(feature = "sqlcipher"))]
+        let db = WalletSqliteDatabase::new(path).await.unwrap();
+
+        // Create a test mint quote
+        let mint_url = MintUrl::from_str("https://example.com").unwrap();
+        let mint_quote = MintQuote {
+            id: "test_quote_id".to_string(),
+            mint_url: mint_url.clone(),
+            payment_method: PaymentMethod::Bolt11,
+            amount: Some(Amount::from(1000)),
+            unit: CurrencyUnit::Sat,
+            request: "test_request".to_string(),
+            state: MintQuoteState::Unpaid,
+            expiry: 1234567890,
+            secret_key: None,
+            amount_issued: Amount::from(0),
+            amount_paid: Amount::from(0),
+        };
+
+        // Add the mint quote
+        db.add_mint_quote(mint_quote.clone()).await.unwrap();
+
+        // Retrieve all mint quotes
+        let quotes = db.get_mint_quotes().await.unwrap();
+
+        // Basic validation
+        assert_eq!(quotes.len(), 1);
+        assert_eq!(quotes[0].id, mint_quote.id);
+    }
+
+    #[tokio::test]
+    async fn test_payment_method_parsing() {
+        use std::str::FromStr;
+        use cdk_common::PaymentMethod;
+
+        // Test standard payment methods
+        assert_eq!(PaymentMethod::from_str("bolt11").unwrap(), PaymentMethod::Bolt11);
+        assert_eq!(PaymentMethod::from_str("bolt12").unwrap(), PaymentMethod::Bolt12);
+
+        // Test case insensitivity
+        assert_eq!(PaymentMethod::from_str("BOLT11").unwrap(), PaymentMethod::Bolt11);
+        assert_eq!(PaymentMethod::from_str("BOLT12").unwrap(), PaymentMethod::Bolt12);
+        assert_eq!(PaymentMethod::from_str("Bolt11").unwrap(), PaymentMethod::Bolt11);
+        assert_eq!(PaymentMethod::from_str("Bolt12").unwrap(), PaymentMethod::Bolt12);
+
+        // Test custom payment methods
+        let custom = PaymentMethod::from_str("lightning").unwrap();
+        assert_eq!(custom, PaymentMethod::Custom("lightning".to_string()));
+
+        let custom_upper = PaymentMethod::from_str("CUSTOM").unwrap();
+        assert_eq!(custom_upper, PaymentMethod::Custom("custom".to_string()));
+    }
```

Comment:
These are good tests but can we move them to where PaymentMethod is defined in nut00.